bin/changelog

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+'use strict';
+
+/*
+ * This script generates the template a changelog by comparing a current version
+ * with master. Run this, copy what's logged into the `CHANGELOG.md` and update
+ * the top section based on the changes listed in "Community Contributions"
+ *
+ * Usage:
+ *
+ * bin/changelog
+ */
+
+var EOL       = require('os').EOL;
+var multiline = require('multiline');
+var Promise   = require('../lib/ext/promise');
+var GitHubApi = require('github');
+
+var github         = new GitHubApi({version: '3.0.0'});
+var compareCommits = Promise.denodeify(github.repos.compareCommits);
+var currentVersion = 'v' + require('../package').version;
+
+compareCommits({
+  user: 'ember-cli',
+  repo: 'ember-cli',
+  base: currentVersion,
+  head: 'master'
+}).then(function(res) {
+  return res.commits.map(function(commitInfo) {
```

Comment:
All that functional programming I've been learning is rubbing off on me. `map.filter.map.sort.map...`